paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -399,5 +397,16 @@ class Paparazzi(
   companion object {
     /** The choreographer doesn't like 0 as a frame time, so start an hour later. */
     internal val TIME_OFFSET_NANOS = TimeUnit.HOURS.toNanos(1L)
+
+    private val isVerifying: Boolean by lazy {
```

Comment:
Probably don't need lazy

Reply:
Actually it's only invoked once below anyway. Could inline.

Reply:
removed lazy